index.bs

```diff
@@ -1274,11 +1274,14 @@ associated.
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialEntity">
     :   <dfn>name</dfn>
     ::  A human-friendly identifier for the entity. For example, this could be a company name for a [=[RP]=], or a
-        user's name. This identifier is intended for display.
+        user's name. This identifier is intended for display. Authenticators while creating a credential MUST support
+        minimum of 64 bytes for this field and optionally can truncate how much it wants to store if this field is more than 64 bytes.
```

Comment:
suggested re-write:
[=Authenticators=] MUST support a 64 byte minimum length for a [=PublicKeyCredentialEntity/name=] members's value. Authenticators MAY truncate a [=PublicKeyCredentialEntity/name=] member's value to a length equal to or greater than 64 bytes.